### ISSUE TITLE
How to keep menu open while navigating to child pages.

### ISSUE THREAD

**Issue by jigs1212** [akveo/nebular#1850]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [ ] bug report
* [ X ] feature request

### Issue description
I am trying to navigate inside menu option and details page, it should keep it open when going inside details page..

**Current behavior:**
The menu collapses

**Expected behavior:**
the menu to be open.

**Steps to reproduce:**

```
export const DASHBOARD_ITEMS: NbMenuItem[] = [
  {
    title: 'Home',
    icon: 'nb-home',
    link: '/home/dashboard/home',
    home: true,
  }];


@Component({
  selector: 'ngx-admin-dashboard',
  template: `
    <ngx-sample-layout>
      <nb-menu [items]="menu" autoCollapse="true"></nb-menu>
      <router-outlet></router-outlet>
    </ngx-sample-layout>
  `,
})



```

example if i hit route '/home/dashboard/home/info' it marks home inactive.

### Other information:

NPM 6.4.1
Node 10.15.1
os linux mint 18.3,
browser chrome.

**Angular 5, Nebular 2.00rc**

**Comment by yggg** [comment]
Hi, @jigs1212. You can try to set the menu item `pathMatch` property to `prefix`. This way if you navigate to the sub route menu item still should be highlighted.
But I'm not sure if this property existed in 2-RC though. I recommend you to update to the latest version as Nebular 2 isn't supported anymore.

**Comment by jigs1212** [comment]
Yes I know about that support is over for 2-rc. And will try to update as we complete our core system. 
Thank you for the `pathMatch` let me try and get back.

**Comment by maihannijat** [comment]
@yggg `pathMatch` property to `prefix` does not work in Nebular 4^.
